client/package.json

```diff
@@ -10,53 +10,53 @@
     "lint:nofix": "vue-cli-service lint --no-fix"
   },
   "dependencies": {
-    "axios": "^0.19.0",
+    "@mdi/font": "^4.5.95",
```

Comment:
To clarify, these are the version this project was already using (per the package-lock.json file). This just makes it easier to compare against other projects.